### DISCUSSION TITLE
React project showing white page

### DISCUSSION THREAD

**Discussion by Gabriel-Moraes-CAD** [community/community#34496] [Other Feature Feedback, Questions, & Ideas]
Hello Github Community!

I published my react project to GitHub pages but while I can see the correct page title and the icon the actual page is blank (nothing shows).
I read through Stackoverflow and GitHub community for solutions. I tried everything I could find but had no success. 

This is my repository: https://github.com/Gabriel-Moraes-CAD/nasa-pro.git
This is my page: https://gabriel-moraes-cad.github.io/nasa-pro/

Thanks in advance for all help!
Cheers!!

**Comment by SilasRodrigues19** [comment]
@Gabriel-Moraes-CAD React is not finding the path because there is no route named "nasa-pro".

![image](https://user-images.githubusercontent.com/47532703/192911078-e578001a-8b8f-42f7-809c-59764e24dfa2.png)

Have you tried correcting the path in package.json, removing "/nasa-pro"?

**Comment by Gabriel-Moraes-CAD** [comment]
@SilasRodrigues19 I tried changing the name and a few more tricks but am not being able to fix this issue.
I also got some suggestions from stack overflow (https://stackoverflow.com/questions/73888210/react-js-project-on-github-pages-not-showing?noredirect=1#comment130466965_73888210) but so far nothing has worked :(

**Comment by Gabriel-Moraes-CAD** [comment]
@SilasRodrigues19 I found part of a solution. instead of using BrowserRouter I used HashRouter. Now I'm working on fixing links on my page. Thanks for your help!

**Reply by SilasRodrigues19** [reply]
@Gabriel-Moraes-CAD I didn't even know about that HashRouter, it seems quite useful in this case. Good job man.

**Reply by Gabriel-Moraes-CAD** [reply]
@SilasRodrigues19 Thank you!
